src/main/java/jalasoft/com/utils/DictionaryUtils.java

```diff
@@ -0,0 +1,25 @@
+package jalasoft.com.utils;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import jalasoft.com.models.Task;
+
+public class DictionaryUtils {
+

```

Comment:
this should have a private constructor, and the methods static

Reply:
Done.